src/SampleExtension/packages.config

```diff
@@ -0,0 +1,10 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="CommonServiceLocator" version="1.0" targetFramework="net45" />
+  <package id="DynamoVisualProgramming.Core" version="2.0.0-beta3907" targetFramework="net45" />
```

Comment:
I thought there is one way to define latest beta with a major version, no?

Reply:
@QilongTang I think you are referring to the dynamoRevit repo which uses sym links, aget, and other trickery. I don't think that works with nuget OOTB.